teammapper-backend/src/jobs/seedMapData.job.ts

```diff
@@ -40,6 +40,7 @@ const createMap = (nodes: IMmpClientNode[]): IMmpClientMap => {
   return {
     uuid: crypto.randomUUID(),
     lastModified: new Date(),
+    lastAccessed: new Date(), // Do we really want to set this here? Without it there's an error
```

Comment:
If I remember correctly it throws an error since it expects all attributes, and lastAccessed isn't optional (in TypeScript's eyes anyway), but if it's only test data we might as well leave it in there.